### PR TITLE
Feature #64

### PR DESCRIPTION
**Pull request by reaby**

### CODE REVIEW

File: eXpansion/Framework/PlayersBundle/Model/Base/Gamecurrency.php [hunk 1]
```diff
@@ -0,0 +1,1601 @@
+<?php
```
Comment (oliverde8) [inline eXpansion/Framework/PlayersBundle/Model/Base/Gamecurrency.php]: Why is this in PlayerBundle ?

File: eXpansion/Framework/PlayersBundle/Model/Base/GamecurrencyQuery.php [hunk 2]
```diff
@@ -0,0 +1,674 @@
+<?php
```
Comment (oliverde8) [inline eXpansion/Framework/PlayersBundle/Model/Base/GamecurrencyQuery.php]: Same why is it here, 

looks like you generated them twice and once in the wrong bundle ?

File: src/eXpansion/Framework/GameCurrencyBundle/Plugins/GameCurrencyPlugin.php [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            $this->entries[$billId] = $entry;
+
+            $this->console->write('$0f0 Success.', true);
+
+            return;
+        } catch (\Exception $e) {
+            $this->console->write(' $f00 Failed. $z'.$e->getMessage());
+        }
+
+        $this->console->write(' $f00 Failed. $z'.$e->getMessage());
```
Comment (oliverde8) [inline src/eXpansion/Framework/GameCurrencyBundle/Plugins/GameCurrencyPlugin.php:126]: We should never come here ?

File: src/eXpansion/Framework/GameCurrencyBundle/Plugins/Gui/BillWindow.php [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            $entries['message']
+        );
+
+        $this->setBusy($manialink, "Processing...");
+
+        if ($bill == false) {
+            $this->closeManialink($manialink);
+        }
+
+        $this->currencyService->sendBill($bill, function () use ($manialink) {
```
Comment (oliverde8) [inline src/eXpansion/Framework/GameCurrencyBundle/Plugins/Gui/BillWindow.php]: You can do better , 
```php
<?php
$this->currencyService->sendBill(
    $bill, 
    function () use ($manialink) {
        $this->closeManialink($manialink);
    }, 
    function () use ($manialink) {
        $this->closeManialink($manialink);
    }
);

File: src/eXpansion/Framework/GameCurrencyBundle/Plugins/GameCurrencyPlugin.php [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+    /**
+     * Set the status of the plugin
+     *
+     * @param boolean $status
+     *
+     * @return null
+     */
+    public function setStatus($status)
+    {
+        // TODO: Implement setStatus() method.
```
Comment (oliverde8) [inline src/eXpansion/Framework/GameCurrencyBundle/Plugins/GameCurrencyPlugin.php]: Not needed remove `StatusAwarePluginInterface`

**Review by oliverde8 — CHANGES_REQUESTED** [review]

File: eXpansion/Framework/PlayersBundle/Model/Base/Gamecurrency.php (hunk 1 above)
Comment (reaby) [reply eXpansion/Framework/PlayersBundle/Model/Base/Gamecurrency.php]: Ooh... i had wrong config at first... i forgot to remove this.

**Review by oliverde8 — APPROVED** [review]

**Pull request merged**